docs/openapi/openapi.yaml

```diff
@@ -219,16 +219,69 @@ components:
     InitialLpa:
       type: object
       required:
+        - type
         - donor
         - attorneys
+        - certificateProvider
+        - signedAt
       properties:
+        type:
+          type: string
+          enum:
+            - pfa
+            - hw
         donor:
           $ref: "#/components/schemas/Donor"
         attorneys:
           type: array
           items:
             $ref: "#/components/schemas/Attorney"
           minLength: 1
+        certificateProvider:
+          $ref: "#/components/schemas/CertificateProvider"
+        peopleToNotify:
+          type: array
+          items:
+            $ref: "#/components/schemas/PersonToNotify"
+        howAttorneysMakeDecisions:
+          type: string
+          enum:
+            - jointly
+            - jointly-and-severally
+            - mixed
+        howAttorneysMakeDecisionsDetails:
+          type: string
+        howReplacementAttorneysMakeDecisions:
+          type: string
+          enum:
+            - jointly
+            - jointly-and-severally
+            - mixed
+        howReplacementAttorneysMakeDecisionsDetails:
+          type: string
+        howReplacementAttorneysStepIn:
+          type: string
+          enum:
+            - all
+            - one
+            - other
+        howReplacementAttorneysStepInDetails:
+          type: string
+        whenTheLpaCanBeUsed:
+          type: string
+          enum:
+            - when-capacity-lost
+            - when-has-capacity
+        lifeSustainingTreatmentOption:
+          type: string
+          enum:
+            - option-a
```

Comment:
These are one of the areas where we have to keep the phrasing at time of signing, i.e. in case the form changes later and the meaning of option-a and option-b change over the product lifetime.

Reply:
Do you mean we'd have to send over "Yes - I do give my attorneys authority to give or refuse consent to life-sustaining treatment on my behalf" / "No - I do not give my attorneys authority to give or refuse consent to life-sustaining treatment on my behalf"?

Reply:
It would make sense if the enum encapsulated what the option means. Does this need to be the whole text? Or could we just have some sort of glossary that the values refer to, and which is pointed to by the schema? The options available might change over time, too, e.g. if some other scenario appeared we haven't thought of: "My attorneys have the option to give or refuse consent to life sustaining treatment on my behalf, providing their decisions are validated by my personal AI doctor first".

Reply:
@hawx yeah, part of this being the "Digital Deed" means we need certain information to be stored in the form it was agreed to at point of signing and prevent drift in meaning over time (i.e. when the labels get changed on the M&R service in 2025 or half way through private beta). 

It might not need to be the full info, but a clearly linked version. Elliot's glossary idea makes sense. I'm almost tempted to use an RDF like approach:

```
<lpa:M-1234> <lpa:lifeSustainingTreatment> <lpa:lifeSustainingTreatmenOptionA-2023-12> 
<lpa:optionA-2023-12> <rdfs:label> "Yes - I do give my attorneys authority to give or refuse consent to life-sustaining treatment on my behalf"
```

Reply:
if the document is described by an overall schema (which we can then document nicely in a human-readable way), I don't think we'd need to go overboard on the options themselves as they would be interpreted in the context of the document's schema

Reply:
We need to come up with a comprehensive plan for this, but I don't think it should block this PR since it'll hold us back a lot. Hopefully wiring it to Sirius will help identify potential issues.

I originally hoped that versioned schemas would encapsulate this information. e.g. if this is schema `//opg/lpa/2023-12-18.json` then we can identify from that what the values were in that "era" of LPA. But thatnow doesn't feel strong enough and it would make any migration difficult. Maybe still helpful for the shape of the LPA.

On the glossary idea: it would be great to publish this in a machine-readable format (e.g. a JSON doc) in this repo so that other codebases can use it as a golden copy. The Sirius frontend could just display `fields.lifeSustainingTreatment[myLstValue].text` rather than have its own calculations or just show "Option A".